token-lending/program/src/processor.rs

```diff
@@ -922,7 +922,7 @@ fn process_repay(
     }
 
     // accrue interest and update rates
-    repay_reserve.accrue_interest(clock.slot)?;
+    assert_last_update_slot(&repay_reserve, clock.slot)?;
```

Comment:
Only because there's no token conversion happening on repay. We just allow borrowers to get the same proportion of collateral for the amount of loan that they pay off at this point.

Reply:
Gotcha, that makes sense, thanks for the explanation!